### ISSUE TITLE
Give time columns (e.g. in backups or logs) a fixed or maximum width

### ISSUE THREAD

**Issue by lognaturel** [getodk/central-frontend#432]
Time columns have a known maximum width. Currently they are responsive just like other columns in the tables meaning they become unnecessarily wide even on not-so-wide screens. It would help with readability if it were fixed or had a maximum width.

Context from comment:
> This came about because I was looking at other systems' logs and those felt a lot tighter to me. I think it's because they either have more columns (e.g. [Zoho](https://www.zoho.com/analytics/help/account-audit-logs.html)) or have a fixed-width table (e.g. [WPEngine](https://wpengine.com/blog/user-portal-activity-log/)). All that had this tighter feeling seemed to have some columns with fixed width (and/or match the content width). Not saying we should do any of those things but that's how I ended up thinking in this space.

**Comment by lognaturel** [comment]
> giving the Time column a fixed px length would work well there

Yes, sounds great! We should do a quick sanity check with other languages and give a bit of extra room.

> consider giving the Type column a fixed px length

Yes, I agree that this also seems like a good candidate. Same comment about other languages.

I agree the App Users table is not a good candidate for making a subset of columns fixed with.

This came about because I was looking at other systems' logs and those felt a lot tighter to me. I think it's because they either have more columns (e.g. [Zoho](https://www.zoho.com/analytics/help/account-audit-logs.html)) or have a fixed-width table (e.g. [WPEngine](https://wpengine.com/blog/user-portal-activity-log/)). All that had this tighter feeling seemed to have some columns with fixed width (and/or match the content width). Not saying we should do any of those things but that's how I ended up thinking in this space.

**Comment by issa-tseng** [comment]
> This came about because I was looking at other systems' logs and those felt a lot tighter to me. I think it's because they either have more columns (e.g. [Zoho](https://www.zoho.com/analytics/help/account-audit-logs.html)) or have a fixed-width table (e.g. [WPEngine](https://wpengine.com/blog/user-portal-activity-log/)). All that had this tighter feeling seemed to have some columns with fixed width (and/or match the content width). Not saying we should do any of those things but that's how I ended up thinking in this space.

it would be really helpful to hear the motivations before/rather than/along with the suggestions! if we start from this observation instead of proposing a fix i can point out that we literally use more space and padding and a larger font size and that's why it feels looser. we can always slim it down if you're looking for a more technical look.

**Comment by lognaturel** [comment]
> really helpful to hear the motivations before/rather than/along with the suggestions!

Mea culpa, I only realized after Matt's response that I had skipped a whole lot of stuff that's only in my head. I do agree that I should always provide context and that there's a broader conversation we can consider One Day™️.

That said, I'm hoping we can consider the cheap change of making a couple columns fixed width. Even on a fairly narrow screen, there's a lot of space after columns that have fixed-width content and then the columns with longer content are tight. Addressing just that would go a long way, I think, and likely even fully resolve my feeling of looseness for this table.
